molecule/app_deploy_standalone/verify.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Verify deployment
+  hosts: all
+  gather_facts: false
+  vars_files:
+    - ../group_vars/all/vars.yml
+  tasks:
+    - name: "Verify WildFly is running"
+      ansible.builtin.uri:
+        url: "http://localhost:{{ wildfly_http_port | default('8080') }}/"
+        method: GET
+        status_code: [200, 404]
+      register: wildfly_status
+
+    - name: "Check if application is deployed via CLI"
+      ansible.builtin.include_role:
+        name: wildfly_utils
+        tasks_from: jboss_cli.yml
+      vars:
```

Comment:
create a `vars.yml` file and put all the vars there and provide it via `vars_files`